conf.py

```diff
@@ -207,21 +207,25 @@
             {
                 'product': '377',
                 'java': '11',
+                'sha256': '7929833412c331fc23cde0e23ca730d652c0be61a8a69c8a82b2af937a3fbd4e',
```

Comment:
Can we rename this to `java_base_image_sha256` or something like that?

Reply:
done https://github.com/stackabletech/docker-images/pull/250/commits/ae7225f1aa0764d15aaf4ab985faabb07593ff2a